services/src/filesystems/virtual_file_system.adb

```diff
@@ -61,9 +63,40 @@ package body Virtual_File_System is
                     Path       : Pathname)
                     return Status_Kind
    is
-      pragma Unreferenced (This, Path);
+      procedure Destroy is new Ada.Unchecked_Deallocation
```

Comment:
Arf, I just noticed that it’s actually a bad idea to do that: it breaks the build of some examples with link errors such as:
`[…]/Ada_Drivers_Library/services/src/filesystems/virtual_file_system.adb:90: undefined reference to `__gnat_free'`

Ok, so the runtime we use for stm32f407disco provides `__gnat_malloc` but it does not provide `__gnat_free`. This leaves me wondering: is it legitimate in this setup to dynamically *allocate* memory?